blog/node-22-ecosystem-update.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Moving our Ecosystem to Node 22
+date: 2024-12-02T00:00:00.000Z
+authors: VerteDinde
+slug: ecosystem-node-22
+tags: [community, ecosystem]
+---
+
+In early 2025, Electron’s npm ecosystem repos (under the `@electron/` and `@electron-forge/` namespaces) will move to Node.js 22 as the minimum supported version.
+
+---
+
+### What does this mean?
+
+In the past, packages in Electron’s npm ecosystem (Forge, Packager, etc) have supported Node versions for as long as possible, even after a version has reached its End-Of-Life (EOL) date. This is done to make sure we don’t fragment the ecosystem—we understand that many projects depend on older versions of Node, and we don’t want to risk stranding those projects unless there was a pressing reason to upgrade.
+
+Over time, maintenance of this policy has become increasingly difficult for a few reasons:
+
+- Lack of official Node.js 14 macOS ARM64 builds requires us to maintain CI infrastructure workarounds to provide full test coverage.
+- `engines` requirements for upstream package dependencies have moved forward, making it increasingly difficult to resolve supply chain security issues with dependency bumps.
+
+Additionally, newer versions of Node.js have included many improvements that we would like to leverage, such as runtime-native common utilities (e.g. [`fs.glob`](https://nodejs.org/api/fs.html#fsglobpattern-options-callback) and [`util.parseArgs`](https://nodejs.org/api/util.html#utilparseargsconfig)) and entire new batteries-included modules (e.g. [`node:test`](https://nodejs.org/api/test.html), [`node:sqlite`](https://nodejs.org/api/sqlite.html)).
```

Comment:
I'm not sure about this one, but should this paragraph be a third bullet point in the previous list? Fits the rule of three